source/module.c

```diff
@@ -828,11 +829,16 @@ PyMODINIT_FUNC PyInit__awscrt(void) {
     }
 
     s_init_allocator();
-    s_install_crash_handler();
 
     /* Don't report this memory when dumping possible leaks. */
     struct aws_allocator *nontracing_allocator = aws_default_allocator();
 
+    struct aws_string *crash_handler_env = NULL;
```

Comment:
Speaking of "AWS_CRT_MEMORY_TRACING", we have [a hack](https://github.com/awslabs/aws-crt-python/blob/ce221859f37325d362c587ce2ef22b5ced3a4502/test/__init__.py#L8-L9) so that it's always turned on when tests run. let's do the same thing and always turn on the crash handler